src/Raven.Studio/typescript/viewmodels/database/documents/editDocument.ts

```diff
@@ -337,14 +337,16 @@ class editDocument extends viewModelBase {
             new getDocumentsFromCollectionCommand(new collection(collectionForNewDocument, this.activeDatabase()), 0, 3)
                 .execute()
                 .done((documents: pagedResult<document>) => {
-                    const schemaDoc = documentHelpers.findSchema(documents.items);
+                    const schemaDoc: any = documentHelpers.findSchema(documents.items);
                     this.document(schemaDoc); 
                     documentTask.resolve(schemaDoc);
                 })
                 .fail(() => documentTask.reject());
         } else {
-            const doc = document.empty();
+            const doc: any = document.empty();
+            doc["Name"] = "...";
```

Comment:
@grisha-kotler Can we avoid doing this? Maybe instead of having name in every created document you can use following bookmarket to bootstrap document with value you want. Just use:
```
javascript:ko.dataFor($("#docEditor")[0]).documentText('Lorem ipsum')
```

Then you will have the content it is useful when demoing with single click.